src/components/Icons/MoreIcon.tsx

```diff
@@ -0,0 +1,45 @@
+import { type IconProps } from './types'
+import { DEFAULT_ICON_COLOR } from './constants'
+
+export function MoreIcon({
+  'aria-label': ariaLabel = 'More Icon',
+  size = 24,
+  color = DEFAULT_ICON_COLOR,
```

Comment:
The color is not propagated to the svg.
I pushed a storybook for the icon, to demonstrate its integrity.